### PR TITLE
Have turbo mode use the rubric template.

### PR DESCRIPTION
**Pull request by dianakhuang**
Add a test to verify that it is working as expected.

A fix for https://openedx.atlassian.net/browse/TNL-3841

Available for review on sandbox: https://dianakhuang.sandbox.edx.org

@andy-armstrong @efischer19

### CODE REVIEW

File: openassessment/templates/openassessmentblock/peer/oa_peer_turbo_mode.html [hunk 1]
```diff
@@ -56,77 +56,7 @@ <h3 class="peer-assessment__display__title">{% trans "Assessment #" %}
...
-                                                    id="peer__assessment__rubric__question--feedback__value"
-                                                    placeholder="{{ rubric_feedback_default_text }}"
-                                                    maxlength="500"
-                                                >
-                                                </textarea>
-                                        </div>
-                                    </li>
-                                </ol>
-                            </fieldset>
+                                {% include "openassessmentblock/oa_rubric.html" with rubric_type="peer" %}
```
Comment (andy-armstrong) [inline openassessment/templates/openassessmentblock/peer/oa_peer_turbo_mode.html]: Sweet! I love replacing huge swaths of code with one line!

File: openassessment/xblock/static/js/spec/lms/oa_peer.js [hunk 2]
```diff
@@ -74,6 +67,17 @@ describe("OpenAssessment.PeerView", function() {
...
         );
+    };
+
+    beforeEach(function() {
+        // Create a new stub server
+        server = new StubServer();
+        server.renderLatex = jasmine.createSpy('renderLatex');
+    });
+
+    it("Sends a peer assessment to the server", function() {
```
Comment (andy-armstrong) [inline openassessment/xblock/static/js/spec/lms/oa_peer.js]: Minor nit: I like Jasmine tests to read like sentences starting with "it", so I lowercase the first letter.

**Comment by andy-armstrong** [comment]
:+1: Nice fix, @dianakhuang!

**Comment by efischer19** [comment]
:+1: Looks good!

**Pull request merged**